DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/EditCentreCustomPromptsAccessibilityTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Web.AutomatedUiTests.AccessibilityTests
+{
+    using DigitalLearningSolutions.Web.AutomatedUiTests.TestHelpers;
+    using FluentAssertions;
+    using Selenium.Axe;
+    using Xunit;
+
+    public class EditCentreCustomPromptsAccessibilityTests : AccessibilityTestsBase
+    {
+        public EditCentreCustomPromptsAccessibilityTests(SeleniumServerFactory<Startup> factory) : base(factory) { }
+
+        [Fact]
+        public void EditRegistrationPrompt_journey_has_no_accessibility_errors()
+        {
+            // Given
+            Driver.LogUserInAsAdminAndDelegate(BaseUrl);
+            const string startUrl = "/TrackingSystem/CentreConfiguration/RegistrationPrompts/1/Edit";
+
+            // When
+            Driver.Navigate().GoToUrl(BaseUrl + startUrl);
+            ValidatePageHeading("Edit delegate registration prompt");
+            var editPageResult = new AxeBuilder(Driver).Analyze();
+
+            Driver.ClickButtonByText("Bulk add");
+            ValidatePageHeading("Configure answers in bulk");
+            var bulkAdditionResult = new AxeBuilder(Driver).Analyze();
+

```

Comment:
Can we also click "Next" on bulk page and then validate that the page heading is "Edit delegate registration prompt again" to ensure the button is correctly configured (given that the IDE doesn't check this automatically)

Reply:
Minor, but we probably want the same for "Add" journey :)